### PR TITLE
docs: load the new examples

### PR DESCRIPTION
**Pull request by vladfrangu**

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "scripts": {
         "examples": "docusaurus-examples",
-        "start": "rimraf .docusaurus && docusaurus start",
+        "start": "rimraf .docusaurus && npm run generate && docusaurus start",
         "build": "rimraf .docusaurus && docusaurus build",
```
Comment (B4nan) [inline website/package.json:5]: i guess we should move the `generate` command here as well (and maybe to `deploy` script too, and remove it from the `docs` workflow)

**Review by B4nan — APPROVED** [review]
otherwise lgtm

File: website/package.json (hunk 1 above)
Comment (vladfrangu) [reply website/package.json]: Agreed, have done that

Comment (B4nan) [reply website/package.json]: now we can also remove the `npm run generate` call from `docs.yml`

Comment (vladfrangu) [reply website/package.json]: Forgot to do that, woops. Committed the removal

**Pull request merged**